### DISCUSSION TITLE
Is duplicate files when exporting with exportTrailingSlash intended?

### DISCUSSION THREAD

**Discussion by davidprae** [vercel/next.js#10989] [Help] [answered]
This post heavily references this closed bug:
https://github.com/zeit/next.js/issues/8119

I found the above GitHub issue when searching for an answer to that question. I implemented the solution mentioned by @devisscher (utilizing exportTrailingSlash) but as @jaredpereira noted, this results in more files than (at least I) expected being created: 

Notably, in my project:
`out/index.html` and  `out/index/index.html`
`out/404.html` and  `out/404/index.html`

<img width="192" alt="Screen Shot 2020-03-12 at 12 41 08 AM" src="https://user-images.githubusercontent.com/12590512/76494312-2ca1da80-63fa-11ea-909d-8c03130433f3.png">

@Timer mentioned something was [intended behavior](https://github.com/zeit/next.js/issues/8119#issuecomment-521645127) but I'm not sure if they were referring to these duplicate files or just that "An index file does not imply a trailing slash".

Is my output described about actually intended? If so, is there a way to get a `/about` type of route without getting duplicate pages?

Thanks!

**Comment by Timer** [comment, marked as answer]
The duplicate files are not intentional! Can you please file a normal bug report? 🙏 

https://github.com/zeit/next.js/issues

**Reply by davidprae** [reply]
Issue created, thanks!

https://github.com/zeit/next.js/issues/11008
